Data.R

```diff
@@ -1,97 +1,131 @@
 #' base R scatterplot matrix
+#+ scatterplot, cache=TRUE
 select(dat0,-'Country/Region') %>% plot()
 #' show first 10 lines of dat0
-head(dat0) %>% pander(split.tables=Inf)
+head(dat0) 
 #' gg plot scatterplot matrix
+#+ ggplot_splom, cache=TRUE
 select(dat0,-'Country/Region') %>% ggpairs()
 #' how to assign value to object and variable
 foo <- 3
 #' How to select columns 2 through 5:
-dat0[,c(2:5)] %>% head %>% pander(split.tables=Inf)
+dat0[,c(2:5)] %>% head
 #' How to select columns by name:
-dat0[,c('Long','Confirmed','Date')] %>% head %>% pander(split.tables=Inf)
+dat0[,c('Long','Confirmed','Date')] %>% head 
 #' This is equivalent to...
 
-head(dat0[,c('Long','Confirmed','Date')])%>% pander(split.tables=Inf)
+head(dat0[,c('Long','Confirmed','Date')])
 #' To see what type each column is
-sapply(dat0, class) %>% pander(split.tables=Inf)
+sapply(dat0, class) %>% pander
 #' Selecting rows instead of columns:
 #'
 #' The old fashioned (base R) way
-dat0[c(1,3,9),] %>% head %>% pander(split.tables=Inf)
+dat0[c(1,3,9),] %>% head()
 #' The subset() command (also base R)
-subset(dat0,Lat<1&Active>0)
+subset(dat0,Lat<1&Active>0) %>% head()
 #' now using OR and equals
-subset(dat0,(Lat<1&Active>0)|`Country/Region`=="Brazil") %>% pander(split.tables=Inf)
+subset(dat0,(Lat<1&Active>0)|`Country/Region`=="Brazil") %>% head()
 #' using OR and excluding
-subset(dat0,(Lat<1&Active>0)|`Country/Region`!="Brazil") %>% pander(split.tables=Inf)
+subset(dat0,(Lat<1&Active>0)|`Country/Region`!="Brazil") %>% head()
 #' Including 
-subset(dat0,(Lat<1&Active>0)|`Country/Region` %in% c("Chile", "Peru")) %>% pander(split.tables=Inf)
+subset(dat0,(Lat<1&Active>0)|`Country/Region` %in% c("Chile", "Peru")) %>% head()
 #' Exclusion of "in" operator
-subset(dat0,(Lat<1&Active>0)|!`Country/Region` %in% c("Chile", "Peru")) %>% pander(split.tables=Inf)
+subset(dat0,(Lat<1&Active>0)|!`Country/Region` %in% c("Chile", "Peru")) %>% head()
 # vignette('dplyr')(tutorial)
-vignette("dplyr")
-# filter allows you to select a subset of rows in a data frame
-dat0 %>% filter(Confirmed == "1", Active == "1")
-# arrange reorders selected rows
-dat0 %>% arrange(Confirmed, Recovered)
-# puts it in descending order
-dat0 %>% arrange(desc(Lat))
-# slice lets you index rows by their integer location
-dat0 %>% slice(5:10)
-# can use head or tail to start from top or bottom
-dat0 %>% slice_head(n=4)
+#vignette("dplyr")
+#' filter allows you to select a subset of rows in a data frame
+dat0 %>% filter(Confirmed == "1", Active == "1") %>% head
+#' arrange reorders selected rows
+dat0 %>% arrange(Confirmed, Recovered) %>% head
+#' puts it in descending order
+dat0 %>% arrange(desc(Lat)) %>% head
+#' `slice` lets you index rows by their integer location
+dat0 %>% slice(5:10) 
+#' can use `head` or `tail` to start from top or bottom
+dat0 %>% slice_head(n=4) 
 dat0 %>% slice_tail(n=10)
-# slice sample randomly selects rows
+#' `slice_sample` randomly selects rows
```

Comment:
Notice that in a `#' ...` line you can put something in back-quotes (`` ` ``) and it will be styled to look like code